Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -426,6 +427,30 @@ void ImageEditor::set_active_layer(Layer* layer)
     }
 }
 
+bool ImageEditor::add_new_layer_from_selection()
```

Comment:
Can we not use the boolean error pattern for future code, please? It would also make this function a bit cleaner.

Reply:
I had considered using ErrorOr<> type, but didn't want to return any other value so had written it off. Didn't realize you can do ErrorOr<void> :) I've updated this.